sylvia-derive/src/interfaces.rs

```diff
@@ -55,4 +62,29 @@ impl Interfaces {
             })
             .collect()
     }
+
+    pub fn emit_proxy_accessors(&self, mt_app: &Type) -> Vec<TokenStream> {
+        self.iter()
+            .map(|interface| {
+                let ContractMessageAttr { module, .. } = interface;
+                assert!(!module.segments.is_empty());
```

Comment:
`filter_map` with `emit_error` would be good here but honestly this should never be empty as `parse` will fail with `expected identifier` error in the `new` method if `#[messages()]` will be empty so it should be safe to just `unwrap` here

Reply:
If it should never be empty, then use a type system in a way it can never be empty.

For now, I can even believe it would never be empty. But things might change in the future. The assumption would be lost, things would get empty and terrible messages would be reported. It is not a way to go.